template.yaml

```diff
@@ -27,10 +27,6 @@ Parameters:
     Type: String
     Description: Connection port number for your RDS Proxy
     Default: 5432
-  dbRegion:
```

Comment:
Just double checking, is it fine to get rid of the region parameter?

Reply:
yea, its no longer needed as a db connection parameter